src/scheduler.rs

```diff
@@ -43,13 +45,22 @@ impl Scheduler {
             reviewed_date: self.now,
         }
     }
+
+    fn init_seed(parameters: &mut Parameters, current: &Card, now: DateTime<Utc>) {
+        let time = now.timestamp_millis();
```

Comment:
It would confuse the user who reviews the same card in the same date without rating it. They will see the interval varies every time.

Reply:
fixed
https://github.com/open-spaced-repetition/rs-fsrs/blob/bde90bb70d1b23002bc9b048c89c21ffe73e2ad2/src/scheduler.rs#L49-L50